index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Desmond Owusu</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <div class="background"></div>
+    <header class="Header">
+      <div class="headerContent">
+        <a class="Logo" href="#">Desmond Owusu</a>
+        <nav>
+          <ul class="navItems">
+            <li class="navItem"><a href="#">Portfolio</a></li>
+            <li class="navItem"><a href="#">About</a></li>
+            <li class="navItem"><a href="#">Contact</a></li>
+          </ul>
+        </nav>
+        <img src="images/menu.png" alt="" class="Menu" />
+      </div>
+    </header>
+    <main class="main">
+      <div class="mainContent">
+        <h1 class="description">
+          I am Desmond, <br />
+          glad to see <br />
+          you!
+        </h1>
+        <p class="aboutMe">
+          I'm a software developer! I can help you build a product, feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
+        <p class="letsConnect">LET'S CONNECT</p>
+        <div class="networkContent">
+            
+            <a href="https://twitter.com/DesmondOwusuDev"><img src=" ./images/twitter.svg" alt="twitter"></a>
+            <a href="https://www.linkedin.com/in/desmond-owusu-ansah-09274a223/"><img src=" ./images/Linkedin-icon.svg" alt="Linkedin"></a>
+            <a href="https://github.com/Owusu-Desmond"><img src=" ./images/github.svg" alt="github"></a>
+            <a href="#"><img src=" ./images/M-icon.svg" alt="M-icon"></a> 
+            <a href="#"><img src=" ./images/hand-raise.svg" alt="handraise icon"></a> 
+          </div>
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly use a `ul` tag to wrap your list of items like this:
```
<ul><li> <a href=""><img /></a></li></ul>
```